docs/documentation/using-notify.md

```diff
@@ -0,0 +1,48 @@
+To set it locally, run this command in your Terminal, in the root
+folder of your prototype:
+```shell
+   echo NOTIFYAPIKEY=xxxxxxx >> .env
+````   
+(where xxxxxxx is a key you’ve created in Notify). This creates a file 
+named `.env` which contains your API key.
+
+To set it on Heroku, go to the settings page on your app, click
+‘reveal config vars’ and enter NOTIFYAPIKEY and the value of the key
+in the two boxes.
+
+Add this code to the top of routes.js:
```

Comment:
Can we break this file up into sections using headings, to make it clear that this step now needs to be done regardless of whether you're running it locally or on Heroku?